src/enums/ganttRole.ts

```diff
@@ -0,0 +1,14 @@
+export enum GanttRole {
+    Legend = "Legend",
+    Task = "Task",
+    Parent = "Parent",
+    StartDate = "StartDate",
+    EndDate = "EndDate",
+    Duration = "Duration",
+    Completion = "Completion",
+    Resource = "Resource",
+    Milestones = "Milestones",
+    // Renaming to "Tooltips" in capabilities.json breaks the report after updating to the newer version
```

Comment:
What is this?

Reply:
Forgot to delete a comment. Essentially, I tried to rename "GanttRole.ExtraInformation" to "GanttRole.Tooltips" to match a display name in capabilities.json, but then found that it's a breaking change. When replacing old version with newer version data field "Tooltips" could not be matched and will be reset.

I deleted the comment.